### PR TITLE
Change in filters showing previous filters data

### PR DESCRIPTION
**Pull request by ManojPathak1**
**Description of PR**
Change in filters showing previous filters data

**Type of PR**

- [x] Bugfix
- [ ] New feature

**Relevant Issues**  
N/A

**Checklist**

- [x] Compiles and passes lint tests
- [x] Properly formatted
- [x] Tested on desktop
- [x] Tested on phone

**Screenshots**
Screenshot 2020-04-11 at 6 38 40 PM

After Fix
Screenshot 2020-04-11 at 6 40 01 PM

### CODE REVIEW

**Review by tannakartikey — COMMENTED** [review]
@ManojPathak1 can you update the description of the PR explaining what changes you did and why?

Also, can you rebase the PR?

**Comment by tannakartikey** [comment]
> @ManojPathak1 can you update the description of the PR explaining what changes you did and why?
> 
> Also, can you rebase the PR?

@ManojPathak1 I just saw the screenshots so ignore the description part.

File: src/utils/common-functions.js [hunk 1]
```diff
@@ -115,6 +115,17 @@ export function sliceTimeseriesFromEnd(timeseries, days) {
...
+/**
+ * Returns whether the value is empty or not.
+ * @param {any} value
+ *
+ * @return {Boolean}
+ */
+export const isEmpty = (value) =>
+  Object.keys(value).length === 0 && value.constructor === Object;
+
+const numberFormatter = new Intl.NumberFormat('en-IN');
```
Comment (tannakartikey) [inline src/utils/common-functions.js:127]: @ManojPathak1 what is the purpose of this?

File: src/utils/common-functions.js [hunk 2]
```diff
@@ -115,6 +115,17 @@ export function sliceTimeseriesFromEnd(timeseries, days) {
   return timeseries.slice(timeseries.length - days);
 }
 
+/**
+ * Returns whether the value is empty or not.
+ * @param {any} value
```
Comment (tannakartikey) [inline src/utils/common-functions.js:120]: Should not we say: `@param {Object} value`

**Review by tannakartikey — CHANGES_REQUESTED** [review]

**Comment by jeremyphilemon** [comment]
Patched by #1363

**Pull request closed without merging**